### PR TITLE
Fully support imaged object id

### PR DESCRIPTION
**Pull request by Bronson-Brown-deVost**
I saw that I failed to fully implement the imaged object id.  It is fixed now along with some bugfixes on the route names for artefacts.

### CODE REVIEW

File: data-access/Models/ArtefactModels.cs [hunk 1]
```diff
@@ -7,10 +7,8 @@ public class ArtefactModel
         public string Mask { get; set; }
         public string TransformMatrix { get; set; }
         public short ZIndex { get; set; }
```
Comment (Bronson-Brown-deVost) [inline data-access/Models/ArtefactModels.cs:9]: We don't need Institution, CatalogNumber1, or CatalogNumber2 in the artefact model, the ImagedObjectId will suffice.

File: sqe-http-api/Controllers/ArtefactController.cs [hunk 2]
```diff
@@ -57,7 +57,7 @@ public async Task<ActionResult<ArtefactListDTO>> GetArtefacts([FromRoute] uint e
         /// <param Name="ArtefactId">Unique Id of the desired artefact</param>
         /// <param Name="images">Defines whether image references should be included in the response</param>
         [AllowAnonymous]
-        [HttpGet("editions/{editionId}/artefacts/{ArtefactId}")]
+        [HttpGet("editions/{editionId}/artefacts/{artefactId}")]
```
Comment (Bronson-Brown-deVost) [inline sqe-http-api/Controllers/ArtefactController.cs:60]: Somehow the route parameter artefactId was changed to ArtefactId by my fixing of the classes to use PascalCase.  I fix it here.

File: sqe-http-api/DTOs/ImagedObject.cs [hunk 3]
```diff
@@ -20,43 +20,5 @@ public class ImagedObjectDTO
     public class ImagedObjectListDTO
     {
         public List<ImagedObjectDTO> imagedObjects { get; set; }
-    };
-
-    public static class ImagedObjectIdFormat
```
Comment (Bronson-Brown-deVost) [inline sqe-http-api/DTOs/ImagedObject.cs:25]: As agreed, we no longer need this function, and it should not have been in the DTO anyway.  It is removed now.

File: data-access/Models/ArtefactModels.cs (hunk 1 above)
Comment (zmbq) [reply data-access/Models/ArtefactModels.cs:9]: Yes, you are correct. Can you please open an issue to Talya show she removes them from the frontend DTO?

Comment (Bronson-Brown-deVost) [reply data-access/Models/ArtefactModels.cs:9]: No problem, I just did it.  You can merge this PR now.

**Review by zmbq — APPROVED** [review]

**Pull request merged**